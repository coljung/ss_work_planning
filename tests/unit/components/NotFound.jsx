import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NotFound from '../../../app/components/NotFound';

describe('NotFound', () => {
    it('should render correctly', () => {
        const output = shallow(
            <NotFound >404!</NotFound>
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
