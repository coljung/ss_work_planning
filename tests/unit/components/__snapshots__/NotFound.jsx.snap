// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<div>
  <h1>
    404!
  </h1>
  <p>
    Page not found
  </p>
</div>
`;
